src/main/java/bio/terra/janitor/db/TrackedResource.java

```diff
@@ -9,15 +14,24 @@
  * table in the Janitor's database.
  */
 @AutoValue
+@JsonSerialize(as = TrackedResource.class)
+@JsonDeserialize(builder = AutoValue_TrackedResource.Builder.class)
```

Comment:
I don't think we should serialize the whole TrackedResource class into the FlightParamters. The cleanup flights don't need to know things like creation/expiration, or the TrackedResourceState that it starts in.
I think we should make a dedicated POJO class that we can serialize as JSON for CleanupFlights. That also allows us to add things to diverge our standard cleanup flight parameters from what's in TrackedResourceState. Keeping serialization (and Stairway counts as a serialization) as a separate layer from our internal representation class (TrackedResource) is a good place to start.

Reply:
I think just CloudResourceUid is enough for a flight to cleanup the resource. So just use the cloudResourceUid instead?

Reply:
Added CleanupParams